README.md

```diff
@@ -102,12 +102,16 @@ If you use the argument `--audio -`, PiFmAdv reads audio data on standard input.
 sox -t mp3 http://www.linuxvoice.com/episodes/lv_s02e01.mp3 -t wav -  | sudo ./pi_fm_adv --audio -
```

Comment:
This exact use case is described, here, why should it be included twice?

Reply:
Yes, but it only specifies **pre recorded** mp3 files, not mentioning support for live mp3s.

Reply:
This has nothing to do with the project though, you can change the wording so it's clear sox supports reading mp3 streams, however there should be no duplicates.

Reply:
Alright, I will as soon as possible.